engine/include/cubos/engine/assets/bridges/json.hpp

```diff
@@ -26,32 +26,26 @@ namespace cubos::engine
     {
     public:
         /// @brief Constructs a bridge.
-        ///
-        /// If the @p indentation level is set to -1, all whitespace is removed.
-        ///
-        /// @param indentation Indentation level to use when saving the JSON file.
-        JSONBridge(int indentation = 4)
+        JSONBridge()
             : FileBridge(core::reflection::reflect<T>())
-            , mIndentation{indentation}
         {
         }
 
     protected:
         bool loadFromFile(Assets& assets, const AnyAsset& handle, core::memory::Stream& stream) override
         {
             // Dump the file stream into a string and initialize a JSON deserializer with it.
-            std::string json{};
-            stream.readUntil(json, nullptr);
-            core::data::old::JSONDeserializer deserializer{json};
+            std::string jsonStr;
+            stream.readUntil(jsonStr, nullptr);
+            core::data::JSONDeserializer deserializer{};
+
+            // New JSONDeserializer() receives a JSON object to deserialize from
+            nlohmann::json json = nlohmann::json::parse(jsonStr);
```

Comment:
Be careful, we need to handle invalid JSON files gracefully. You should catch the thrown exception and return false (check `engine/src/cubos/engine/scene/bridge.cpp:44` for an example).